db/connect.js

```diff
@@ -1,12 +1,13 @@
-const mongoose = require('mongoose')
+const mongoose = require('mongoose');
 
 const connectDB = (url) => {
-  return mongoose.connect(url, {
-    useNewUrlParser: true,
-    useCreateIndex: true,
-    useFindAndModify: false,
-    useUnifiedTopology: true,
-  })
+  // return mongoose.connect(url, {
+  //   useNewUrlParser: true,
+  //   useCreateIndex: true,
+  //   useFindAndModify: false,
+  //   useUnifiedTopology: true,
+  // })
+  return mongoose.connect(url);
```

Comment:
Just curious if there was a reason you commented these out? Does the instructor also do so or were you having any errors with the options that were in there?

Reply:
Yes, I do get this error

![image](https://github.com/hammaazarok/06-jobs-api/assets/19716151/64ceea47-3b7d-4572-b80c-73f7c9890d1f)

[https://stackoverflow.com/questions/68958221/mongoparseerror-options-usecreateindex-usefindandmodify-are-not-supported](https://stackoverflow.com/questions/68958221/mongoparseerror-options-usecreateindex-usefindandmodify-are-not-supported)

![image](https://github.com/hammaazarok/06-jobs-api/assets/19716151/f13da5f2-14a3-412f-888a-10b422e20217)

Reply:
Aha okay got it 👍  You're using mongoose v7 and the instructor is using mongoose v5. That's where the difference comes in. Okay what you have looks good then!